<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorMatInput</class>
 <widget class="QDialog" name="MorMatInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingMorphologyMat</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cboxPreview">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>preview</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>31</y>
     <width>238</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sboxWidth">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sboxHeight">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cboxSquared">
      <property name="text">
       <string>squared</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cboxType">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>rectangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ellipse</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>pressed()</signal>
   <receiver>MorMatInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>pressed()</signal>
   <receiver>MorMatInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
